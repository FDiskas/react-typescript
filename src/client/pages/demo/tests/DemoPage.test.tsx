import * as React from 'react';
import { shallow } from 'enzyme';
import enzymeToJson from 'enzyme-to-json';

import { DemoPage } from '../DemoPage';

describe('DemoPage.js', () => {
    it('renders without crashing', () => {
        shallow(<DemoPage/>);
    });

    it('matches snapshot', () => {
        const wrapper = shallow(<DemoPage/>);
        expect(enzymeToJson(wrapper)).toMatchSnapshot();
    });

});
