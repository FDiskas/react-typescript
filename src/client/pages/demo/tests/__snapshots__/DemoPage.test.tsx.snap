// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoPage.js matches snapshot 1`] = `
<main
  className="page"
>
  Component name: DemoPage
  <Connect(App)
    addAction={[Function]}
    items={
      Array [
        Object {
          "id": "",
          "label": "",
          "value": "",
        },
      ]
    }
  />
</main>
`;
